src/buza/static/buza/css/questions.css

```diff
@@ -8,10 +8,15 @@
     margin-top: 1em;
     margin-bottom: 1em;
     /* Shadow */
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.15);
```

Comment:
removed the card shadow because it makes it difficult to spot the highlight on hover